src/sensors/distance/ultrasound/ping/SR04.cpp

```diff
@@ -43,16 +43,14 @@ unsigned int SR04::getDistance()
 
     // Generate the pulse
     mRuntime.setPinState(kTriggerPin, kLow);
-    static constexpr auto kSetPinToKnownStateDelay = 5;
-    mRuntime.delayMicros(kSetPinToKnownStateDelay);
+    mRuntime.delayMicros(5); // Set pin to known state
     mRuntime.setPinState(kTriggerPin, kHigh);
-    static constexpr auto kTenMicrosecondPulse = 10;
-    mRuntime.delayMicros(kTenMicrosecondPulse); // 10us pulse
+    mRuntime.delayMicros(10); // 10us pulse
     mRuntime.setPinState(kTriggerPin, kLow);
     // Wait for the pulse to arrive and measure its duration
     auto duration = mRuntime.getPulseDuration(kEchoPin, kHigh, kTimeout);
     // Calculate how much far out the object is
-    auto calculatedDistance
+    unsigned int calculatedDistance
```

Comment:
:warning: **modernize-use-auto** :warning:
use auto when initializing with a cast to avoid duplicating the type name
```suggestion
    auto```